apps/src/studio/studio.js

```diff
@@ -5653,7 +5653,7 @@ Studio.setSprite = function(opts) {
 
   var spriteIndex = opts.spriteIndex;
   if (spriteIndex < 0 || spriteIndex >= Studio.spriteCount) {
-    throw new RangeError('Incorrect parameter: ' + spriteIndex);
```

Comment:
Does the student see anything indicating the mistake they made when this error is thrown currently?

Reply:
Great question.  No, the student experience didn't change from my testing.  Both before and after, the program would just stop running with the offending block highlighted.  But before, we'd be firing an exception every frame.